<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="GGN000179422001" name="Selenium Grid2" tests="2" failures="1" timestamp="27 May 2019 13:58:44 GMT" time="1.335" errors="0">
  <testcase name="eightyTwenty" time="0.008" classname="TestCases.TestReRun"/>
  <testcase name="testbugFromJenkins" time="0.01" classname="TestCases.TestReRun">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
Another soft assert failure">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Another soft assert failure
	at TestCases.TestReRun.testbugFromJenkins(TestReRun.java:26)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- testbugFromJenkins -->
</testsuite> <!-- Selenium Grid2 -->
